src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -316,7 +316,7 @@ public static bool IsLinuxConsumptionOnAtlas(this IEnvironment environment)
                    string.IsNullOrEmpty(environment.GetEnvironmentVariable(LegionServiceHost));
         }
 
-        private static bool IsLinuxConsumptionOnLegion(this IEnvironment environment)
+        public static bool IsLinuxConsumptionOnLegion(this IEnvironment environment)
```

Comment:
I'm using this existing helper to determine when we're CV1 on Legion. Need to verify that this is still correct.